virttest/virsh.py

```diff
@@ -3329,6 +3329,33 @@ def snapshot_list(name, options=None, **dargs):
     return ret
 
 
+def snapshot_status(name, options=None, **dargs):
```

Comment:
I wonder if we should focus on virsh utilities in virsh.py and move other related functions to utils_libvirt. 

BTW, can you plz give an example what kind of 'status' the snapshots could be? or it should be 'snapshot_state'?

Reply:
hi yingshun ,Added the example

Reply:
@nanli1 I don't mean just adding an example in this comment, it's also about the name and the location, can you plz recheck it?